Python/sampleclient/filestorage.py

```diff
@@ -0,0 +1,73 @@
+from .azure import AzureStorageContainer
+from .filechecks import (file_exists, file_size_in_bytes, files_must_not_be_more_than, MAX_FILE_SIZE_FOR_PUT_API)
+from .error import error_out
+
+
+class FileUpload():
+    def __init__(self, logger_object, config, args, msrd_client):
+        self.log = logger_object
+        self.config = config
+        self.args = args
+        self.msrd = msrd_client
+
+    def upload_files(self, job_archive_path, presubmit_script_path):
+        """
+        Upload files to remote endpoints so that the job submition can reference them via URL.
+        Returns: tuple (job_blob_url, presubmit_blob_url, )
```

Comment:
The exception would be if you had a huge tuple or list, and you were writing it like:
```python
t = (
    'x',
    'y',
    'z',
)
````
We don't have a Python style guide yet, but I think this will be part of it.

Reply:
Per PEP8 - https://www.python.org/dev/peps/pep-0008/#when-to-use-trailing-commas - I was adding the commas because I was trying to be explicit and clear about thier usage.  Technically optional, yes, but with the added "(" and ")" still very explicit and clear.

One thing I'm running into here in terms of style is I normally have a linter/PEP8/PEP20 check running in my IDE for making sure none of my code violates PEP8; but I dont have these tools here since I was told I wouldnt be getting a license for my normal Python IDE, so I'm just making due what I have.  

Should I not be trying to follow PEP8?